web/.eslintrc.json

```diff
@@ -27,6 +27,7 @@
 
     // unnecessary due to prettier
     "function-paren-newline": "off",
+    "indent": "off",
```

Comment:
Why this one?  😛

Reply:
because prettier handles it and sometimes will conflict with eslint's ideas of where indents are "needed"